File: python/ultrasonicSensorRead.py
from megapi import *

def onRead(v):
	print "distance:"+str(v)+" cm";

if __name__ == '__main__':
	bot = MegaPi()
	bot.start("/dev/ttyUSB0")
	while 1:
		sleep(0.1);
		bot.ultrasonicSensorRead(6,onRead);

File: Libs/NodeForMegaPi/examples/ultrasonicSensorRead.js
var MegaPi = require("../megapi").MegaPi;
var bot = new MegaPi("/dev/ttyS0",onStart);

function onRead(dist){
  console.log(dist+" cm");
}

function loop(){
  bot.ultrasonicSensorRead(6,onRead);
}
function onStart(){
  setInterval(loop,300);
}
